src/modules/auth/views/AuthLoginView.vue

```diff
@@ -0,0 +1,43 @@
+<script setup lang="ts">
+import { ref } from 'vue'
+import { useI18n } from 'vue-i18n'
+import { useRouter } from 'vue-router'
+
+import AuthLoginForm from '@/modules/auth/components/AuthLoginForm.vue'
+import { useAuthStore } from '@/modules/auth/stores/auth.store'
+
+const { t } = useI18n()
+const authStore = useAuthStore()
+const router = useRouter()
+
+const isIncorrect = ref<boolean>(false)
+
+async function handleLogin(data: { username: string; password: string }): Promise<void> {
+	try {
+		await authStore.login(data)
+	} catch (e) {
+		isIncorrect.value = true
+	}
+
+	router.push({ name: 'todos' })
+}
+</script>
+
+<template>
+	<div class="flex h-screen w-screen flex-col items-center justify-center">
+		<div>
+			<h1 class="w-[360px] text-3xl font-semibold">{{ t('auth.login.welcome') }}</h1>
```

Comment:
van deze kan je ook components maken, bv `AppTitle` en `AppText`. Dan kan je al die styling daar in wegsteken